### PR TITLE
Add file detection options

### PR DESCRIPTION
**Pull request by killjin**
- [x] Changes have been thoroughly tested to not break existing functionality.
- [x] New functionality has been documented or existing documentation has been updated to reflect changes.
- [x] Visual changes are explained in the PR description using a screenshot/recording of before and after.

image

This option speeds up the form access speed. Trying to modify js is unsuccessful. If it is 0 here, it is better to show that the file size is unavailable.

### CODE REVIEW

**Comment by killjin** [comment]
Is there a way to return 0 or -1 or false, hide the file size display, or display as Size not available, for example:

filepond Labels: labelFileSizeNotAvailable
DEFAULT VALUE: 'Size not available'
Label used when no file size information was received

File: packages/forms/src/Components/BaseFileUpload.php [hunk 1]
```diff
@@ -79,7 +81,9 @@ protected function setUp(): void
             $files = collect(Arr::wrap($state))
                 ->filter(static function (string $file) use ($component): bool {
                     try {
-                        return blank($file) || $component->getDisk()->exists($file);
```
Comment (killjin) [inline packages/forms/src/Components/BaseFileUpload.php:82]: here should be 
```php
filled($file) && $component->getDisk()->exists($file);
```

**Comment by Nuhel** [comment]
```php
 $FileDetection  = $component->shouldFileExistDetection();
```
 
 Variable name should started with lowercase latter.

**Comment by danharrin** [comment]
Hi, thanks for the PR. I think it's important to give you a bit of information as to why this PR has been tagged as "low priority". We really appreciate all contributions, and intend to visit this in the future, and hopefully merge it if it is a suitable feature for the framework.

However, reviewing PRs, and often rewriting them on behalf of the author to ensure they are consistent with the rest of the framework, consumes a lot of my time. At the moment we have a large backlog of bug reports that I am prioritising, since most of these affect functionality that worked in a previous version and are now impacting real projects. Hopefully you can see how that work comes first before introducing new features that people aren't using in projects yet.

Again, thank you for contributing and I'm looking forward to reviewing this PR when our backlog of bug reports has reduced.

**Comment by danharrin** [comment]
Thanks

**Pull request merged**